.github/workflows/multiple-build.yaml

```diff
@@ -81,9 +82,9 @@ jobs:
         id: push
         uses: ./
         with:
-          image: ${{ secrets.REGISTRY_USER }}/${{ env.IMAGE_NAME }}
+          image: ${{ secrets.REGISTRY_USER }}
```

Comment:
I think this should be `${{ env.IMAGE_NAME}}`

Reply:
yes, haha